### DISCUSSION TITLE
Creating Supercell Thunderstorms

### DISCUSSION THREAD

**Discussion by okienative99** [niels747/2D-Weather-Sandbox#78] [Show and tell]
![s1](https://user-images.githubusercontent.com/120753895/227605544-9c315573-ae84-4477-9b1c-65565566e548.JPG)
![s2](https://user-images.githubusercontent.com/120753895/227605546-b03eb52a-8ce0-4503-8c7d-31ea62567017.JPG)
![s3](https://user-images.githubusercontent.com/120753895/227605547-4e14a1b7-31eb-4b56-8a4e-2f0c5ae8d1fa.JPG)
![s4](https://user-images.githubusercontent.com/120753895/227605548-c380d5bc-aaa2-4efc-8823-5fcea39e6585.JPG)

Hi! 

So, I think I figured out the recipe for creating supercells with "tornadoes." I've been working on this for a while until I got the perfect settings to create powerful updrafts with strong CAPE values. The storms will start to fire off once the wind speeds up and the dewpoint temps increase. You will start to see dust getting kicked up before the storms form. This typically happens in the Great Plains where I live when strong southwesterly winds increase from the west and merge with the moisture from the Gulf of Mexico. Though this is a 2-D map, I believe this simulated a very close severe weather-style set up. 

Once the updrafts start to fire, you will see hail forming then falling in a downdraft which then balances out the updraft creating a "tornado" from the dust being kicked up. Since the upper level winds are moving the storms fast, the updraft will stay sustained for a longer period of time before the downdraft overpowers it. Then creating new storms to fire off on a cold boundary created from the dying storm. 

**The map:** 
You must ensure you have a 'gulf' or large body of water somewhere on the map. I like to add mine to the far left and land on the right. The land must be completely flat to sustain powerful updrafts with no disruptions. Make sure to have plenty of vegetation with some dry spots to simulate drought and allow for dust to be picked up. (This will get sucked into the updraft causing a tornado.)

Control settings below:

**Fluid**
Voriticy: 0.01. 
Wind: 0.19. 
Global heating: -0.0001

**Radiation**
Latitude: 34 (Oklahoma sits at this latitude. A state known for powerful tornadoes.)
Month: 5 (May)
Sun Intensity: 2 
IR Multiplier: 0

**Water**
Lake/Sea temp: 26.7
Land Evaporation: max
Lake/Sea Evaporation: max
Evaporation Heat: max
Water weight: max

**Precipitation**
Spawn rate: max
Snow Density: 0.16
Fall speed: max
Growth Rate 0c: 0.001
Growth Rate -30c: max
Freezing rate: max 
Melting rate: max  
Evaporation rate: max

**Comment by okienative99** [comment]
PS: Sometimes it will create a Squall Line style event when the conditions are right!

**Reply by odvigajlo** [reply]
WOW thanks for sharing those settings! I already tried them, and those are best storms i seen in game.

**Reply by okienative99** [reply]
Of course! I've lived in Oklahoma all my life and studied the weather growing up. Once I played the simulation for a while, I figured out the perfect conditions for creating a favorable environment---thanks to the T-Skew graph as a guide. :) 

Regarding controls, feel free to play around and toggle the settings a bit, but you can use this strategy as a fundamental base. The faster the wind, the more discreet the supercells are! Just like in real life.

**Reply by CheetahCyberBoi** [reply]
I got a scenario where a supercell would form, then when it dies it would sent out a cold front to then spark new convection and create another.

**Reply by coolkid0281** [reply]
I would love to see how you did it

**Reply by Gooberman22334** [reply]
i really need a tutorial on this

**Reply by okienative99** [reply]
I can make one for YouTube! I've been thinking about making videos for a while now and now is a good opportunity to do so. Lol

**Comment by Davi678967** [comment]
instructions unclear: i created a hurricane

**Reply by Davi678967** [reply]
it created a 318 kmph gust

**Reply by Davi678967** [reply]
413 now

**Reply by Davi678967** [reply]
700+ after i put the water temp at 40 and put it all on water

**Comment by that1guyontheinternet** [comment]
it doesnt work. am i doing smth wrong?
